calculate_average_agoncal.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# sdk use java 21.0.1-tem
+
+JAVA_OPTS="--enable-preview -XX:+UseShenandoahGC -XX:+UseStringDeduplication -da"
```

Comment:
To use Shenandoah, please add a prepare script which enables an SDK build which provides that GC. You can find an example in _repare_twobiers.sh_ (which uses Temurin).

Reply:
Done with https://github.com/gunnarmorling/1brc/pull/460/commits/10b3e62a268754ba5740d41230125ee0bbe4fb2b